faqs.md

```diff
@@ -0,0 +1,155 @@
+
+### What should I know about the GMN programme?
+Designed for any Galaxy members (both new and existing), this two months Gmentoring programme will equip participants (Gmentees and Gmentors) with a broad set of practical skills to boost knowledge and thrive in the Galaxy community.
+
+**Gmentees** can be any Galaxy member. During the course of the program, Gmentees are required to complete their individual or team projects and share them with their Gmentor. 
+
+Gmentees must be willing to accept constructive feedback from their Gmentors related to their projects and goals and commit themselves to continuous growth and improvement in project skills and knowledge in order to develop new skills and expertise and to be proficient and grow with Galaxy.
+
+**Gmentors** are seasoned Galaxy community members who can commit to regular meetings for 2 months (recommended: at least 30 mins meetings biweekly). Gmentors help to ensure their Gmentee has a successful transition into the Galaxy community. 
+
+They serve as a resource and repository of information, connect with people and the community, assist in purposeful networking, and provide thought and guidance throughout the completion of the project.
+
+**Network Managers** are the programme coordinators. They help with pairing Gmentors with Gmentees, sending mail to both Gmentors and Gmentees to inform them about their acceptance into the program, monitoring the progress of the Gmentorship, collecting feedback, and ensuring Gmentor and Gmentee complete the program. They are also the mediator when either Gmentor or Gmentee has any challenges during the process.
+
+This Galaxy Gmentor Network Programme helps participants (network managers, Gmentors, Gmentees) to:
+- share their expertise and develop skills and knowledge vital to create, lead, and contribute to Galaxy projects.
+- connect with members across different projects, communities, backgrounds, and identities.
+- empower each other to become effective Galaxy community members.
+
+### How to join the program?
+**As part of the process:**
+
+Gmentees join this program with goals that they want to achieve during this program. With support from their mentors, they complete these successfully.
+
+Gmentors join the program by indicating their availability and willingness to mentor someone.
+
+Network Managers accept to be part of the program by replying to the invite message.
+
+### Can you tell me more about the Gmentors-Gmentees-Pairing (MMP) Process?
+Once a Gmentee submitted an application, it will be reviewed by the network manager and then shared with the pool of possible Gmentors. The Gmentors select their own Gmentees together with the network manager. The areas of Gmentorship are not limited to: Contributing to the community, Running data analysis, Instructing students, and Carrying out projects. This opportunity will help them over a period of two months to develop new skills and expertise and to be proficient and grow with the ecosystem.
```

Comment:
```suggestion

Once a Gmentee submitted an application, it will be reviewed by the network manager and then shared with the pool of possible Gmentors. The Gmentors select their own Gmentees together with the network manager. The areas of Gmentorship are not limited to: Contributing to the community, Running data analysis, Instructing students, and Carrying out projects. This opportunity will help them over a period of two months to develop new skills and expertise and to be proficient and grow with the ecosystem.
```